kbase-extension/static/kbase/js/common/html.js

```diff
@@ -1,20 +1,13 @@
-/*global
- define
- */
-/*jslint
- browser: true,
- white: true
- */
 define([
     'underscore',
     'uuid'
-], function (underscore, Uuid) {
+], (underscore, Uuid) => {
```

Comment:
Most of the changes in this file come from running ESLint and Prettier; I will comment on any manual changes I made.